File: spark16/src/main/java/scasql/LoadSave.scala
package scasql

import org.apache.spark.sql.SQLContext
import org.apache.spark.{SparkConf, SparkContext}

/**
  * Created by 小省 on 2016/7/1.
  */
object LoadSave {
  def main(args: Array[String]) {
    val conf =new SparkConf().setAppName("LoadSave").setMaster("local[4]")
    val sc =new SparkContext(conf)
    val sqlContext=new SQLContext(sc)

    val DF=sqlContext.read.load("users.parquet")
    DF.printSchema()
    DF.show()
    // 'overwrite', 'append', 'ignore', 'error'.
    DF.write.mode(saveMode = "Error").format("json").save("a")

    sc.stop()
  }

}

File: spark16/src/main/java/javasql/LoadSave.java
package javasql;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;
import org.apache.spark.sql.SQLContext;


/**
 * Created by 小省 on 2016/7/1.
 */
public class LoadSave {
	public static void main(String[] args) {
		SparkConf conf =new SparkConf().setAppName("LoadSave").setMaster("local[4]");
		JavaSparkContext sc =new JavaSparkContext(conf);
		SQLContext sqlContext =new SQLContext(sc);
//		DataFrame df= sqlContext.read().parquet("users.parquet");
////				.load("users.parquet");
//		df.show();
//		df.write().mode(SaveMode.Overwrite).save("nan");
//		df.save("na", SaveMode.Overwrite);

		sqlContext.sql("select * from text.students.txt").show();
		
		sc.stop();


	}
}
